solax/inverter.py

```diff
@@ -54,32 +57,46 @@ async def get_data(self):
         return data
 
     @classmethod
-    async def make_request(cls, host, port, pwd="", headers=None):
+    async def make_request(cls, host, port, pwd="", headers=None) -> InverterResponse:
         """
         Return instance of 'InverterResponse'
         Raise exception if unable to get data
         """
         raise NotImplementedError()
 
     @classmethod
-    def sensor_map(cls):
+    def sensor_map(cls) -> Dict[str, Tuple[int, Measurement]]:
         """
         Return sensor map
         """
         sensors = {}
-        for name, (idx, unit, *_) in cls.response_decoder().items():
+        for name, mapping in cls.response_decoder().items():
+            unit = Measurement(Units.NONE)
+
+            if isinstance(mapping, tuple):
+                (idx, unit_or_measurement, *_) = mapping
+            else:
```

Comment:
This `else` clause is never hit - coverage is complaining at me in #77 about this line never executing. A grep finds no inverter subclasses that don't give units for values.

Reply:
OK - this code does use this for the "resets" values, which I've removed in 77. Sorry for false alarm